person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = generate_id
```

Comment:
- [ ] Kindly define `generate_id` to generate the ID, or you can simply generate it quickly.

  ```
  @id = rand(1...1000)
  ```